meta-xfstk/conf/layer.conf

```diff
@@ -0,0 +1,10 @@
+# We have a conf and classes directory, add to BBPATH
+BBPATH := "${BBPATH}:${LAYERDIR}"
+
+# We have a recipes-* directories, add to BBFILES
+BBFILES := "${BBFILES} ${LAYERDIR}/recipes-*/*/*.bb \
+	${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "xfstk"
+BBFILE_PATTERN_xfstk = "^${LAYERDIR}/"
+BBFILE_PRIORITY_xfstk = "10"
```

Comment:
LAYERSERIES_COMPAT_xfstk = "rocko sumo thud warrior zeus"

(I only checked zeus)

Reply:
Done! I have added all of them for now, once we get the build going on zeus I can test the others.